test/cases/api.sh

```diff
@@ -641,6 +694,10 @@ test $((INIT_COMPOSES+1)) = "$SUBS_COMPOSES"
 # shellcheck disable=SC2024
 sudo jq -rM .args.manifest /var/lib/osbuild-composer/jobs/"${COMPOSE_ID}".json > "${ARTIFACTS}/manifest.json"
```

Comment:
@achilleas-k Hm, since composer is backed by a DB here, I assume we want a sql query here. Should I just dump the `args jsonb`?

Reply:
Wait, I guess args.Manifest only :) nvm,  think I found it :D